### DISCUSSION TITLE
Unable to login to web3auth via React Native Firebase auth

### DISCUSSION THREAD

**Discussion by hyetigran** [Web3Auth/Web3Auth#1059] [Help]
Seeing the following error message:
`Unable to detect login share from the Auth Network`

I'm using `@web3auth/react-native-sdk` v3.3.0

Here is the relevant snippet of code:
```
export const loginWeb3Auth = async (idToken: string) => {
  try {
    const web3auth = new Web3Auth(WebBrowser, {
      clientId: WEB3_AUTH_CLIENT_ID,
      network: WEB3_AUTH_NETWORK,
      loginConfig: {
        jwt: {
          name: 'hedgehog-polls',
          verifier: 'hedgehogpolls-firebase',
          typeOfLogin: 'jwt',
          clientId: WEB3_AUTH_CLIENT_ID,
        },
      },
    });

    let loginOptions = {
      loginProvider: LOGIN_PROVIDER.JWT,
      redirectUrl: 'hhpolls://auth',
      mfaLevel: MFA_LEVELS.NONE,
      curve: 'ed25519' as SUPPORTED_KEY_CURVES_TYPE,
      extraLoginOptions: {
        id_token: idToken,
        verifierIdField: 'uid',
        domain: 'http://localhost:3000',
      },
    };

    const info = await web3auth.login(loginOptions);

    console.log('INFO INFO\n', info.userInfo);
    console.log('INFO INFO KEY\n', info.privKey);
    return info;
  } catch (error) {
    console.log('Error', error);
    throw new Error('Fail to initialize with Web3Auth');
  }
};
```

![Snip20221212_3](https://user-images.githubusercontent.com/5153613/207143289-97cab0c1-6893-44de-ab81-32b9e7d402ca.png)

device info for additional detail:
System:
    OS: macOS 12.5
    CPU: (10) arm64 Apple M1 Pro
    Memory: 94.88 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.18.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 8.19.2 - /usr/local/bin/npm
    Watchman: 2022.11.28.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 16.0, macOS 12.3, tvOS 16.0, watchOS 9.0
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.3 AI-213.7172.25.2113.9123335
    Xcode: 14.0.1/14A400 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.17 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.1.0 => 18.1.0 
    react-native: 0.70.5 => 0.70.5 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

**Comment by steoniy** [comment]
same issue off fresh github clone using my verifier

**Reply by cryptowampum** [reply]
We have the same problem

**Comment by urgn** [comment]
Try using firebase app id as client id in JWT.

**Reply by hyetigran** [reply]
example shows otherwise:
https://github.com/Web3Auth/examples/blob/main/react-native/rn-bare-firebase-example/App.js#L54-L64
